hr_holidays_public/models/hr_employee.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployeeBase(models.AbstractModel):
+    _inherit = "hr.employee.base"
+
+    is_public_holiday = fields.Boolean(
+        string="Public Holiday Today", compute="_compute_is_public_holiday"
+    )
+
+    def _compute_is_public_holiday(self):
+        holiday_public = self.env["hr.holidays.public"]
+        for item in self:
+            item.is_public_holiday = holiday_public.is_public_holiday(
+                fields.Date.today(), employee_id=item.id
```

Comment:
Please use `context_today` to avoid issues with negative UTC offsets.
```suggestion
                fields.Date.context_today(item), employee_id=item.id
```